pages/api/hello.js

```diff
@@ -1,8 +1,8 @@
-import { fromUrl } from "hosted-git-info";
-import fetch from "node-fetch";
+import { fromUrl } from 'hosted-git-info';
+import fetch from 'node-fetch';
```

Comment:
Hey @thinklikeadesigner, I'd rather not do this as part of this diff. Not sure if this is intentional change also. DO you prefer single quotes over double?

In any case would make it a different change if feel strongly about it

Reply:
air bnb style guide is single quotes, so that is what my eslint is set at

Reply:
That's cool to set it up this way. Please make this change as a separate PR. Don't do it as part auth diff.
Also please make sure it is part of this project's eslint config. It shouldn't be part of local/dev env preference. I will be happy to help you with that. It should be separate change.